tests/unit/loginTest.js

```diff
@@ -9,6 +9,11 @@ import React from 'react';
 import renderer from 'react-test-renderer';
 import App from '../../src/App';
 
+jest.mock('@react-native-firebase/crashlytics', () => ({
+    log: jest.fn(),
+    recordError: jest.fn(),
+}));
+
```

Comment:
This test started to fail because as a `native` test it imports the `ErrorBoundary/index.native.js` file and that file tries to load `@react-native-firebase/crashlytics`

Reply:
Sounds good, can you leave a comment above it just to clarify that?